roles/monitoring/defaults/main.yaml

```diff
@@ -16,6 +16,7 @@ V4M_MONITORING_NAMESPACE: monitoring
 V4M_KIBANA_FQDN: "dashboards.{{ V4M_BASE_DOMAIN }}"
 V4M_KIBANA_CERT: "{{ V4M_CERT }}"
 V4M_KIBANA_KEY: "{{ V4M_KEY }}"
+V4M_KIBANA_LOGADM_PASSWORD: "{{ V4M_KIBANA_PASSWORD if V4M_KIBANA_PASSWORD else lookup('password', '/dev/null chars=ascii_letters,digits') }}"
```

Comment:
Looking for clarification. This lookup actually runs the `password` command with the parameters passed to generate the password if `V4M_KIBANA_PASSWORD` is not set.

Reply:
That is correct.